src/ui/public/courier/courier.js

```diff
@@ -29,33 +29,36 @@ import '../promises';
 
 import { requestQueue } from './_request_queue';
 import { FetchSoonProvider } from './fetch';
-import { SearchLooperProvider } from './search_looper';
+import { SearchPollProvider } from './search_poll';
 
 uiModules.get('kibana/courier').service('courier', ($rootScope, Private) => {
   const fetchSoon = Private(FetchSoonProvider);
 
   // This manages the doc fetch interval.
-  const searchLooper = Private(SearchLooperProvider);
+  const searchPoll = Private(SearchPollProvider);
 
   class Courier {
     constructor() {
       // Listen for refreshInterval changes
       $rootScope.$listen(timefilter, 'refreshIntervalUpdate', function () {
```

Comment:
@nreese I'm surprised to find instances of `$listen(timefilter` in the code after #19852. These should all be updated to use `$listenAndDigestAsync(timefilter` right?

Reply:
I never set courier listen to listenAndDigestAsync since the callback does not modify any scope variables.

Reply:
Hmm, well this one does: https://github.com/elastic/kibana/blob/5f0e590ad5812fd503b3b0fbfbff6cafb67bbda5/src/core_plugins/kibana/public/discover/controllers/discover.js#L339-L341. I think it's probably safer to just convert everything to `$listenAndDigestAsync(timefilter`...